From ee459ee801ec3359bd82d2742eefdfa516ad1a16 Mon Sep 17 00:00:00 2001
From: Caleb Cushing <xenoterracide@gmail.com>
Date: Thu, 17 Jun 2010 09:47:28 -0400
Subject: [PATCH] limit no strict to refs only

Signed-off-by: Caleb Cushing <xenoterracide@gmail.com>
---
 Timer.pm |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Timer.pm b/Timer.pm
index c803055..b5502ff 100644
--- a/Timer.pm
+++ b/Timer.pm
@@ -65,7 +65,7 @@ our $epoch = undef;
 our @totals;
 
 foreach my $sub ( qw( process include ) ) {
-    no strict;
+    no strict 'refs';
     my $super = __PACKAGE__->can("SUPER::$sub") or die;
     *{$sub} = sub {
         my $self = shift;
-- 
1.7.1

